### DISCUSSION TITLE
Sequence if more than 1 WiFi station exists?

### DISCUSSION THREAD

**Discussion by mugu57** [nakhonthai/ESP32APRS_T-TWR#39] [Q&A]
Hello 

I do have two sites at which my T-TWR Plus should operate, main home and my holiday site.
In order to easier use the configuration I configure in the Wireless Menu under WiFi Multi station the following Wireless stations, see screenshot below.

  
![grafik](https://github.com/nakhonthai/ESP32APRS_T-TWR/assets/147989973/a399accc-9855-461c-8af8-710f5b62998c)

WiFi station# 1 is my home and works, in other words if I power up the T-TWR Plus within some seconds logs into the WiFi with SSID UL3Fritz.
WiFi station#3 (which is my holiday site during weekend) never works it does not log into my router at the holiday site.
I tried to switch on and off several times but it did not log into the WiFi router.
Could you explain if my understanding is wrong that it try’s at time of booting all three enabled sites in a sequence? Do I have to disable the two other stations #1 and #2 to enable station#3 site?

Thank you for your help and assistance 
Vy 73 de Michael DB2GM

**Comment by nakhonthai** [comment]
You can activate them together.
The wifi will connect. When the scan finds that SSID in list and RSSI to high.
